db_management/db_online.rst

```diff
@@ -55,3 +85,36 @@ database.
 
     .. image:: media/dup_expires.png
         :align: center
+
+.. _delete_online_database:
+
+Deleting a Database
+===================
+
+You can only delete databases of which you are the Administrator.
+
+.. danger:: When you delete your database all the data will be permanently lost.
+    The deletion is instant and for all the Users. We strongly suggest to backup
```

Comment:
one more empty line to create a real line return in rst.
`users` lowercase.
I thought Odoo was making backup. What happens to them?
"suggest **you** to backup" ?